### DISCUSSION TITLE
Academic Conference 2023 books

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#236] [General]
Academic conference 2022 workshop때 2023년에 진행할 책을 선정하였습니다.

- #197 

그래서 결정된 책은 아래와 같습니다.

|종필|동훈|태형|
|-----|-----|-----|
|![image](https://user-images.githubusercontent.com/17442457/185042164-b2a9e9cf-d121-45b6-bcae-1b23c13e834c.png)|![image](https://image.aladin.co.kr/product/29464/92/cover500/k422837236_1.jpg)|![image](https://user-images.githubusercontent.com/17442457/185042665-468f3a33-6a7e-4a44-868e-9acebb27acb2.png)|

## Academic Conference 2023 일정입니다.

- Home comingday 2022 행사가 열리는 12월 17일 부터 멤버 모집시작, 12월 29일 까지 모집
- 2022-12-30 금요일 오후 11시, 오리엔테이션 진행 (멤버 소개, 토론 진행 방식 소개, 책 읽는 순서 정하기 등)
- 2022-12-30 부터 2023-06-16 까지, 총 sprint 12를 진행. 1 sprint == 2 weeks
- 2023-06-23 금요일, Academic conference 2023 workshop 진행

**Comment by TaeHyoungKwon** [comment]
![books](https://user-images.githubusercontent.com/18538418/185197996-ae64685b-dd00-4bf4-bbf7-1fb4412c4e71.jpg)

선구매 해두었습니다 ㅎㅎ

**Reply by jongfeel** [reply]
책은 구매해 둔게 있을 거라는 생각은 했는데 
전부 구매하고 인증샷까지 ㅋㅋㅋ
역시 대단하십니다!
